core/frontend/src/ContextRealityModelState.ts

```diff
@@ -77,6 +78,13 @@ export class ContextRealityModelState {
   public get appearanceOverrides(): FeatureAppearance | undefined { return this._appearanceOverrides; }
   public set appearanceOverrides(overrides: FeatureAppearance | undefined) { this._appearanceOverrides = overrides; }
   public get modelId(): Id64String | undefined { return (this._treeRef instanceof RealityTreeReference) ? this._treeRef.modelId : undefined; }
+  public get isGlobal(): boolean {

```

Comment:
Is this trying to serve the same purpose as `isContentUnbounded` and if so do we need both?

Reply:
I added a comment.  Global spans the entire global ellipsoid as a 3D tileset.  The existing isUnbounded refers to map that can be projected without boundary in either planar or ellipsoidal projection.